### DISCUSSION TITLE
Tweaks Channel doesn't appear on Slack

### DISCUSSION THREAD

**Discussion by cunhap** [mallowigi/mtslack#81] [Q&A]
**Describe the bug**
After applying tweaks, the buttons appear on the header but the "slack channel" to manage settings does not.

**To Reproduce**
1. Apply Tweaks
2. Restart Slack
3. See error

**Expected behavior**
I should see the Tweaks "channel" on Slack.

**Desktop (please complete the following information):**
 - OS: macOS Big Sur

**Comment by mallowigi** [comment]
which version of slack od you have?

**Comment by cunhap** [comment]
Screen shot of the about menu. 
![image](https://user-images.githubusercontent.com/64083007/124446387-48ac5680-dd78-11eb-8fe2-6a0176c422f4.png)

**Comment by mallowigi** [comment]
Are you sure you ran with `sudo mtslack` ? And where is your Slack app located?

**Comment by cunhap** [comment]
I'm pretty sure! :D 

Slack is located inside /Applications. Could this be the reason? Does it need to be inside my home's Applications folder?

**Comment by mallowigi** [comment]
No, it should be working. Do you get an error when running it ?

**Reply by cunhap** [reply]
No, no errors whatsoever. Is there a way to increase log level or to add more debug information?
